src/node/rpc/frontend.h

```diff
@@ -213,7 +213,11 @@ namespace ccf
     std::optional<std::vector<uint8_t>> process_command(
       std::shared_ptr<enclave::RpcContext> ctx,
       Store::Tx& tx,
-      CallerId caller_id)
+      CallerId caller_id,
+      std::function<
+        void(Store::Tx& tx, enclave::RpcContext& ctx, RpcFrontend& frontend)>
```

Comment:
Can we please name this type with a `using`, rather than repeating it? And I believe this default lambda can be replaced with `= {}`.